plugins/pulp_rpm/plugins/distributors/yum/publish.py

```diff
@@ -871,7 +871,7 @@ def process_main(self):
         """
         Call out to createrepo command line in order to process the files.
         """
-        pipe = subprocess.Popen('createrepo -d --update --skip-stat %s' % self.content_dir,
+        pipe = subprocess.Popen('createrepo_c -d --update --keep-all-metadata --skip-stat %s' % self.content_dir,
```

Comment:
this is >100 lines, its probably easier to fix it now than to fix when merging to master